Sources/Kitura/Router.swift

```diff
@@ -726,8 +733,11 @@ extension Router : HttpServerDelegate {
             sendResourceIfExisting(routeResp, resource: "index.html")
         } else {
             do {
-                try routeResp.status(HttpStatusCode.NOT_FOUND).send("Cannot \(String(routeReq.method).uppercased()) \(routeReq.parsedUrl.path ?? "").").end()
-            } catch {}
+                let message = "Cannot \(String(routeReq.method).uppercased()) \(routeReq.parsedUrl.path ?? "")."
+                try routeResp.status(HttpStatusCode.NOT_FOUND).send(message).end()
```

Comment:
If this pull request will follow IBM-Swift/Kitura-net#22, which will probably happen, I'll fix it :)